host/tests/integration/components/catalog-entry-editor-test.gts

```diff
@@ -16,6 +16,13 @@ class MockLocalRealm extends Service {
   url = new URL(testRealmURL);
 }
 
+class MockLoaderService extends Service {
```

Comment:
we shouldnt need this is we just manipulate the loader from the real loader service